packages/strapi-tiptap-editor/admin/src/components/Editor/index.tsx

```diff
@@ -71,10 +73,10 @@ const Editor = ({ editor, settings, productPrice }: EditorProps) => {
           productPrice={productPrice}
         />
         <BubbleMenuComponent editor={editor} />
-
-        <Box padding={2} background="neutral0" maxHeight="600px" style={{ resize: 'vertical', overflow: 'auto' }}>
-          <EditorContent editor={editor} />
-        </Box>
+        <EditorContent
+          editor={editor}
+          className={currentTheme === 'dark' ? 'utrecht-strapi-editor-content--dark' : undefined}
+        />
```

Comment:
Ik stel voor om het ongeveer zo te doen:

Zie: [`utrecht-document--surface` class name](https://github.com/nl-design-system/utrecht/blob/main/components/document/src/_mixin.scss#L32)

```tsx
<Document className={clsx('utrecht-document--surface', { 'utrecht-theme--dark' : currentTheme === 'dark' })"> {/* zorgt voor background-color enzo */}
  <RichTextEditorCanvas>
    <EditorContent editor={editor} />
  </RichTextEditorCanvas>
</Document>
```

```css
.utrecht-rich-text-editor-canvas {
    min-block-size: var(--utrecht-rich-text-editor-min-block-size);
    padding-block-end: var(--utrecht-rich-text-editor-canvas-padding);
     padding-block-start: var(--utrecht-rich-text-editor-canvas-padding);
     padding-inline-end: var(--utrecht-rich-text-editor-canvas-padding);
     padding-inline-start: var(--utrecht-rich-text-editor-canvas-padding);
    resize: vertical; /* fallback needed perhaps, not sure about browser support for block */
    resize: block;
}
```

de tokens dan in de `.utrecht-theme` zetten